client/on-demand-entries-client.js

```diff
@@ -3,14 +3,20 @@
 import Router from '../lib/router'
 import fetch from 'unfetch'
 
+const {
+  __NEXT_DATA__: {
+    assetPrefix
```

Comment:
Should this be `assetPrefix = ''` to use empty string in case `window.__NEXT_DATA__.assetPrefix` is undefined?